nsv13/code/modules/overmap/aim_helper.dm

```diff
@@ -0,0 +1,107 @@
+/mob/living
+	var/obj/structure/overmap/gunning_ship = null //NSV13 - reference to overmap for aim helper
```

Comment:
Not hugely sure how I feel about having another overmap reference further to get_overmap()

Reply:
I got rid of it